tests/test_contribution_summary.py

```
@@ -67,12 +68,25 @@ def test_list_contribution_summaries_counts_and_scores(tmp_path) -> None:
-    # Merge-only scoring: bob has 1 merged PR, so score is 4 (pr_merged weight)
-    assert bob.total_score == 4
+    assert bob.total_score == 0
+
+
+def test_list_contribution_summaries_rejects_deprecated_scoring_args(tmp_path) -> None:
+    storage = SqliteStorage(str(tmp_path))
+    storage.init_schema()
+    period_end = datetime(2024, 1, 31, tzinfo=timezone.utc)
+    period_start = period_end - timedelta(days=30)
+
+    with pytest.raises(ValueError):
```

Ruff (0.15.18)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)